src/@carbon-info/views/Ecosystem/components/AppsCarousel.tsx

```diff
@@ -222,6 +212,29 @@ const useStyles = makeStyles((theme: Theme) => ({
     fontWeight: 700,
     marginTop: "1rem",
   },
+  itemContainer: {
+    height: '100vh',
+    display: 'flex',
+    alignItems: 'center',
+    [theme.breakpoints.down('sm')]: {
+      height: '55vh',
```

Comment:
also need to reduce the height for mobile to reduce the gap

<img width="266" alt="image" src="https://github.com/Switcheo/carbon-website/assets/88943108/c343ccfe-198a-4864-9be5-be10734b6ab8">